config/locales/forms.yml

```diff
@@ -317,6 +321,36 @@ en:
         reason_for_leaving: Reason for leaving
         salary: Salary scale or level
         subjects: Subjects and key stages taught
+      jobseekers_job_application_details_qualifications_category_form:
+        category_options:
+          gcse: GCSE
```

Comment:
alhpabetical order

Reply:
The order in this file is determined by the order that we want them to be displayed on the form.